server/plugin/plugin.go

```diff
@@ -642,7 +642,11 @@ func (p *Plugin) storeGitHubToUserIDMapping(githubUsername, userID string) error
 
 func (p *Plugin) getGitHubToUserIDMapping(githubUsername string) string {
 	var data []byte
-	_ = p.client.KV.Get(githubUsername+githubUsernameKey, data)
+	err := p.client.KV.Get(githubUsername+githubUsernameKey, &data)
+	if err != nil {
+		p.API.LogError("Error occurred while getting the user ID from KV store using the Github username", "Error", err.Error())
```

Comment:
It's not actionable for an admin.
```suggestion
		p.API.LogWarn("Error occurred while getting the user ID from KV store using the Github username", "Error", err.Error())
```

Reply:
@hanzei Updated